collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/DeleteSavedFormTest.java

```diff
@@ -36,4 +38,18 @@ public void deletingAForm_removesFormFromFinalizedForms() {
                 .pressBack(new MainMenuPage())
                 .assertNumberOfFinalizedForms(0);
     }
+
+    @Test
+    public void accessingSortMenuInDeleteSavedInstancesShouldNotCrashTheAppAfterRotatingTheDevice() {
```

Comment:
I was hoping I could say "this could be a test for `XFragment`, but deleting a saved form still uses `AppListFragment` sadly! Hopefully this kind of test can be moved down to a Fragment level when we make the same changes we made for deleting blank forms (creating `DeleteBlankFormFragment` for example).